source/indexes/create-search-index.txt

```diff
@@ -0,0 +1,67 @@
+.. _compass-create-search-index:
+
+============================
+Create an Atlas Search Index
+============================
+
+You can create Atlas Search indexes in |compass|. Atlas Search indexes
+let you query data in `Atlas Search
+<https://www.mongodb.com/docs/atlas/atlas-search/>`__. Atlas Search
+indexes enable performant text search queries by mapping search terms to
+the documents that contain those terms.
+
+About this Task
+---------------
+
+To create an Atlas Search index, your deployment must be hosted on
+`MongoDB Atlas <https://www.mongodb.com/docs/atlas/>`__ and have a
+cluster tier of M10 or higher.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+   
+   .. step:: Open the index creation dialog
+
+      From the :ref:`Indexes <collection-tab>` tab, click the
```

Comment:
I think this ref should be to the indexes tab:

```suggestion
      From the :ref:`Indexes <indexes-tab>` tab, click the
```

Reply:
Ah, thanks! Fixed. And also fixed on the procedure used on the main indexes procedure.